.github/workflows/commit-message-check.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
       uses: tim-actions/commit-message-checker-with-regex@v0.3.1
       with:
         commits: ${{ steps.get-pr-commits.outputs.commits }}
-        pattern: '^.+(\n.{0,72})*$|^.+\n\s*[^a-zA-Z\s\n]|^.+\n\S+$'
+        pattern: '^[a-zA-Z]+(\n.{0,72})*$|^[a-zA-Z]+\n\s*[^a-zA-Z\s\n]|^[a-zA-Z]+\n\S+$'
```

Comment:
I don't understand the pattern. From https://github.com/GsActions/commit-message-checker, it looks like the pattern includes the title of the commit. So you are testing `[a-zA-Z]` on that title, and asking it to consist of only alpha characters before the `\n`? It looks wrong.

I would try the following pattern:

```
^[a-zA-Z].{0,71}(\n([a-zA-Z].{0,71}|[^a-zA-Z\n].*|))+$
```

(Tested using https://regex101.com, seems to give the correct results)

Reply:
The `.+` before the `\n` is used to match the title, so we can't replace it with [a-zA-Z] for titles always include colons and numbers

Reply:
The second part of the regex `^.+\n\s*[^a-zA-Z\s\n]` is for the long lines support, `\s*[^a-zA-Z\s\n]` will match non-alphabetic.